sqlmesh/core/config/connection.py

```diff
@@ -2067,6 +2116,9 @@ class RisingwaveConnectionConfig(ConnectionConfig):
     pre_ping: bool = True
 
     type_: t.Literal["risingwave"] = Field(alias="type", default="risingwave")
+    dialect: t.ClassVar[t.Literal["risingwave"]] = "risingwave"
+    display_name: t.ClassVar[t.Literal["RisingWave"]] = "RisingWave"
+    display_order: t.ClassVar[t.Literal[16]] = 16
```

Comment:
[Nit] I suggest to make class variables all caps to follow the convention for class constants